### PR TITLE
ESLint Configuration

### PR DESCRIPTION
**Pull request by JosteinLindhom**

### CODE REVIEW

File: public/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
+	"parser": "@typescript-eslint/parser",
+	"plugins": ["@typescript-eslint"],
+	"extends": [
+		"eslint:recommended",
+		"plugin:react/all"
+    ],
```
Comment (meling) [inline public/.eslintrc.js:7]: This line seems to have spaces, while all others use tab. When I save this file, this line changes those spaces to tab. Not sure what config is doing that though.

**Review by meling — CHANGES_REQUESTED** [review]
Check my suggestions!

**Comment by meling** [comment]
I guess I've already reviewed this, but it is still in draft mode. More to be added to it?

File: public/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (JosteinLindhom) [reply public/.eslintrc.js:1]: It does for me as well. However, converting it to an ES module causes ESLint to stop working. Some solutions are mentioned [here](https://stackoverflow.com/questions/48624588/use-export-default-in-eslintrc-js-file-instead-of-module-exports), but none seem to be worth doing.

Alternatively we could simply rename `.eslintrc.js` -> `.eslintrc.json` and skip the `module.exports =` part entirely.

**Review by meling — APPROVED** [review]

**Pull request merged**